assets/terraform-modules/bootkube/assets.tf

```diff
@@ -31,26 +31,26 @@ resource "local_file" "bootstrap-scheduler" {
 resource "local_file" "kube-apiserver" {
   filename = "${var.asset_dir}/charts/kube-system/kube-apiserver.yaml"
```

Comment:
Can you please rebase the PR, I think we have fixed this

Reply:
Will cherry-pick as it has not been merged yet.